### PR TITLE
✨ Metrics: add simcore_user_agent in service_started/service_stopped metric

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

add the simcore_user_agent label onto the service_started/service_stopped metrics on the webserver.

**Please note that it only works with dynamic sidecar powered services!!** Currently I do not have a solution for computational ones. @elisabettai 

image

## Related issue/s
relates with #3908

### CODE REVIEW

**Review by elisabettai — APPROVED** [review]
Thanks a lot for the quick fix!

Comp. services are not that urgent at the moment, at least for the NIH metrics. Maybe at some point we'll need that for s4l-ralated services (e.g. isolve).

Ah @sanderegg, I've also just noticed in the screenshot one thing.
For dy-sidecar-powered services `simcore_user_agent="undefined"` it means that was an actual user (not a puppet), while for comp services we just have it "undefined" (for users as well as puppets).

If I'm understanding this well it might be a bit confusing, but probably there's no easy fix for this.

File: packages/models-library/src/models_library/rabbitmq_messages.py [hunk 1]
```diff
@@ -85,7 +85,8 @@ class InstrumentationRabbitMessage(RabbitMessageBase, NodeMessageBase):
     service_type: str
     service_key: str
     service_tag: str
-    result: Optional[RunningState] = None
+    result: RunningState | None = None
+    simcore_user_agent: str
```
Comment (pcrespov) [inline packages/models-library/src/models_library/rabbitmq_messages.py:89]: Q: is there a specific pattern to identify these agents (i.e. someting that can be more restrictive than ``str``)?

File: services/web/server/src/simcore_service_webserver/projects/projects_handlers_crud.py [hunk 2]
```diff
@@ -147,6 +150,9 @@ async def create_projects(request: web.Request):
         query_params=query_params,
         predefined_project=predefined_project,
         product_name=req_ctx.product_name,
+        simcore_user_agent=request.headers.get(
```
Comment (pcrespov) [inline services/web/server/src/simcore_service_webserver/projects/projects_handlers_crud.py:153]: I need to make sure I do not forget tho add these headers in my PR

https://github.com/ITISFoundation/osparc-simcore/blob/bf8fcec519d02f4b03aaad04e2309907e30df443/api/specs/webserver/scripts/openapi_projects_crud.py#L62-L81

**Review by pcrespov — APPROVED** [review]
I think it is better if I leave this PR going before  https://github.com/ITISFoundation/osparc-simcore/pull/4064

NOTE: do not bother adding headers in the OAS since i will have to redo them

File: packages/models-library/src/models_library/rabbitmq_messages.py (hunk 1 above)
Comment (sanderegg) [reply packages/models-library/src/models_library/rabbitmq_messages.py:89]: currently there is only one of these. and that is only for puppeteer.
https://github.com/ITISFoundation/osparc-simcore/blob/580f666e01341a7d4d50f0c292c3f999d067c150/tests/e2e/tutorials/tutorialBase.js

**Pull request merged**